src/AlertsManagement/AlertsManagement/ActionRuleCommands/SetAzureAlertProcessingRule.cs

```diff
@@ -677,7 +677,7 @@ private void ValidateSchedule()
             {
                 if (System.DateTime.TryParseExact(ScheduleStartDateTime, format, CultureInfo.InvariantCulture, DateTimeStyles.None, out outDateTime) == false)
                 {
-                    throw new PSInvalidOperationException("Invalid ScheduleStartDateTime Format");
+                    throw new PSInvalidOperationException("Invalid ScheduleEndDateTime Format");
```

Comment:
Line 678 is checking ScheduleStartDateTime, why was this 680 throwing error about ScheduleEndDateTime?

Reply:
The if statement at line 676 validates the ScheduleEndDateTime parameter

Reply:
ScheduleStartDateTime on line 678 has been updated to ScheduleEndDateTime